client/user/src/mealorder/menu/meal-order-menu-order.html

```diff
@@ -0,0 +1,199 @@
+<link rel="import" href="../../../bower_components/iron-pages/iron-pages.html">
+<link rel="import" href="../shell/redux-store.html">
+<link rel="import" href="../../../bower_components/polymer/lib/elements/dom-repeat.html">
+<link rel="import" href="../menu/meal-order-vendor-menus.html">
+<dom-module id="meal-order-menu-order">
+    <template>
+        <style>
+            body {
+                font-family: 'Roboto', 'Noto', sans-serif;
+                margin: 0;
+                display: flex;
+                align-items: flex-end;
+                height: 80px;
+                background: #009688;
+                color: white;
+            }
+
+            paper-tabs {
+                -webkit-font-smoothing: antialiased;
+                width: 100%;
+                margin-bottom: 1px;
+                --paper-tabs-selection-bar-color: #179EFF;
+                --paper-tab-ink: #179EFF;
+            }
+
+            paper-tabs {
+                margin: 3% auto;
+                width: 70%;
+            }
+
+            paper-tab.iron-selected .day-of-the-week {
+                color: #179EFF;
+            }
+
+            paper-tab.iron-selected .day {
+                color: #179EFF;
+            }
+
+            .link {
+                @apply --layout-horizontal;
+                @apply --layout-center-center;
+                text-decoration: none;
+                color: #4f4f4f;
+                font-size: 1.5em;
+            }
+
+            .day-of-the-week {
+                font-size: 0.7em;
+                font-weight: lighter;
+            }
+
+            @media only screen and (max-width: 600px) {
+
+                paper-tabs {
+                    margin-top: 3%;
+                    width: 100%;
+                }
+
+            }
+        </style>
+        <app-location route="{{route}}"
+                      use-hash-as-path></app-location>
+
+        <app-route
+                route="{{route}}"
+                pattern="/:menu"
+                data="{{routeData}}">
+        </app-route>
+
+        <paper-tabs class="tabs" selected="0">
+            <template is="dom-repeat" items="{{menuForDates}}" as="calendar">
+                <paper-tab link>
+                    <a class="link"
+                       href="/#/menu/#/[[calendar.date.year]]/[[calendar.date.month]]/[[calendar.date.day]]"
+                       on-click="_handleClick">
+                        <div>
+                            <div class="day-of-the-week">Пн</div>
+                            <div class="day">{{calendar.date.day}}</div>
+                        </div>
+                    </a>
+                </paper-tab>
+            </template>
+        </paper-tabs>
+
+        <meal-order-vendor-menus items="[[menu]]" on-dish-added="addDishToOrder"></meal-order-vendor-menus>
+
+    </template>
+    <script>
+        /**
+         * menu-order is container for vendors' menus and navigation.
```

Comment:
`meal-order-menu-order`. It's strange that `container for vendors' menus and navigation` is called `menu-order.